modules/exploits/windows/misc/allmedia_1_6_buffer_overflow.rb

```diff
@@ -0,0 +1,89 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+# Author: Hejap Zairy
+# Date: 1.08.2022
+# Exploit Prof
+# Proof and Exploit:
+#image:https://i.imgur.com/yLrRR2t.png
+#video:https://streamable.com/x4i50c
```

Comment:
```suggestion
```
All of this information normally goes into the description fields, and it seems you already filled these fields out with the appropriate information. We don't allow this type of information to go elsewhere so I'm going to have to remove this.